dd-sdk-android/src/main/kotlin/com/datadog/android/rum/internal/domain/scope/RumViewManagerScope.kt

```diff
@@ -72,6 +75,22 @@ internal class RumViewManagerScope(
 
     // region Internal
 
+    @WorkerThread
+    internal fun startApplicationLaunchView(event: RumRawEvent, writer: DataWriter<Any>) {
```

Comment:
Can be `private` I think

```suggestion
    private fun startApplicationLaunchView(event: RumRawEvent, writer: DataWriter<Any>) {
```